packages/desktop-gui/src/settings/node-version.jsx

```diff
@@ -0,0 +1,96 @@
+import _ from 'lodash'
+import { observer } from 'mobx-react'
+import React from 'react'
+
+import ipc from '../lib/ipc'
+
+const openHelp = (e) => {
+  e.preventDefault()
+  ipc.externalOpen('https://on.cypress.io/configuration#Node-Version')
+}
+
+const renderLearnMore = () => {
+  return (
+    <a href='#' className='learn-more' onClick={openHelp}>
+      <i className='fa fa-info-circle'></i> Learn more
+    </a>
+  )
+}
+
+const NodeVersion = observer(({ project }) => {
+  const { resolvedConfig, resolvedNodeVersion, resolvedNodePath } = project
+
+  const usesSystemNodeVersion = !!resolvedNodePath
+
+  function formatPluginsFile () {
+    const pluginsFile = _.get(resolvedConfig, 'pluginsFile.value')
+
+    if (pluginsFile) {
+      return <span><code>{pluginsFile}</code></span>
+    }
+
+    return null
+  }
+
+  function formatIntegrationFolder () {
+    const integrationFolder = _.get(resolvedConfig, 'integrationFolder.value')
+
+    return <code>{integrationFolder}</code>
+  }
+
+  return (
+    <div className="node-version">
+      {renderLearnMore()}
+      <div>
+        {
+          usesSystemNodeVersion ?
+            <div>
+              <table className="node-table">
+                <tbody>
+                  <tr>
+                    <th>System Version</th>
+                    <td>
+                      <code>
+                        {resolvedNodeVersion}
+                      </code>
+                    </td>
+                  </tr>
+                  <tr>
+                    <th>
+                      Node.js Path
+                    </th>
+                    <td>
+                      <code>{resolvedNodePath}</code>
+                    </td>
+                  </tr>
+                </tbody>
+              </table>
+              <p className="text-muted node-version-text">
+                <strong>Note:</strong> This version of Node.js was detected on your{' '}
+                <a href="#" onClick={openHelp}>
+                  <code>system</code>
```

Comment:
`system` being in `<code>` doesn't make sense to me. I guess it's supposed to reference that `nodeVersion` can be `system`, but I didn't immediately make that connection.

Reply:
yeah 😬